content/docs/welcome-guide/requirements.md

```diff
@@ -119,3 +115,98 @@ If the current CMake version was not returned because CMake cannot be found, loc
 1. Choose **OK** to save your changes and close the windows that were opened.
 
 1. Verify that `cmake` is on the system path by opening a new command line window and running `cmake --version` again.
+
+## macOS
+
+Support for developing on macOS is in an experimental stage. At a minimum, you need macOS Catalina, Intel x86_64, XCode 11, and a Metal-compatible video card that meets the hardware requirements above.
+
+## Linux 
+
+The primary Linux distribution for using the O3DE editor is Ubuntu 20.04.3 LTS. The following instructions describe how to retrieve and install the required software packages through Ubuntu's **apt-get** command-line utility.
+
+
+### CMake
+
+As with the other operating systems, [CMake {{< versions/cmake >}} or later](https://cmake.org/download/) is required to configure and build O3DE projects. We strongly recommend that you install the **Latest Release** of CMake rather than the default one provided by your current Linux distribution. If CMake is already installed, but does not match the minimum version, you will need to remove it with the following command.
+
+```shell
+sudo apt-get remove cmake
+```
+
+In order to get the latest version of CMake for Ubuntu, you can add the Kitware APT repository to your Ubuntu package list and run **apt-get** to install it. Refer to [Kitware APT Page](https://apt.kitware.com/) for more information.
+
+```shell
+wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | gpg --dearmor - | sudo tee /usr/share/keyrings/kitware-archive-keyring.gpg >/dev/null
+
+echo 'deb [signed-by=/usr/share/keyrings/kitware-archive-keyring.gpg] https://apt.kitware.com/ubuntu/ focal main' | sudo tee /etc/apt/sources.list.d/kitware.list >/dev/null
+
+sudo apt-get update
+
+sudo apt-get install cmake
+```
+
+Once installed, verify that the version meets the minimum version criteria.
+
+```shell
+cmake --version
+```
+
+### Clang
+
+O3DE requires Clang to compile all of the native C++ code. Although Clang-6 is supported, it is recommended to use Clang-12 for the build process.
+
+```shell
+sudo apt-get install clang-12 
+```
+
+### Python 3.7 dependency on libffi
+
+Ubuntu 20.04.3 LTS does not support Python 3.7, however O3DE's local python package is Python 3.7 and has a library dependency on an earlier version of libffi that does not come with Ubunutu 20.04.3 LTS. You will need to manually install the older version of libffi onto Ubunutu 20.04.3 LTS in order for O3DE's Python to execute properly. The steps below demonstrate how to create a temp folder under `/tmp` to download and manually install the specific debian package.
+
+{{< note >}}
+Curl is required to perform the download step shown in the following instructions.
```

Comment:
- Switch to active voice. This gives users clear instruction of what action to take. 
```suggestion
You must install [Curl](https://github.com/curl/curl) to perform the download step shown in the following instructions.
```

Reply:
The URL referenced in this suggestion is for the curl open source. Typically in Linux, a package like Curl is installed with the linux distribution's package manager. For ubuntu, it would be :
```
sudo apt-get install curl
```
I can either remove the URL or add curl in the linux requirements in the welcome page (there is already a list of packages there)